app/models/manageiq/providers/amazon/storage_manager/ebs/refresh_parser_inventory_object.rb

```diff
@@ -59,6 +59,11 @@ def parse_snapshot(snap)
     }
   end
 
+  # Overridden helper methods, we should put them in helper once we get rid of old refresh
+  def get_from_tags(resource, item)
+    (resource['tags'] || []).detect { |tag, _| tag['key'].downcase == item.to_s.downcase }.try(:[], 'value')
+  end
+
```

Comment:
@gberginc - Haha I completely know what you mean. If the RefreshParser wasn't on the cusp of being deprecated i would like to see it addressed, but refactoring this disrupts a number of classes and I'm not sure it is worth the effort when it will be addressed properly for the G release.